lib/config-loader.js

```diff
@@ -1,3 +1,13 @@
 var configPath = process.env.CONFIG_PATH || '../config';
-module.exports = require(configPath);
+const config = require(configPath);
+module.exports  = config;
 
+config.repos = config.repos.map(normalizeRepo);
```

Comment:
This seems verbose for avoiding ES6 sake?
```suggestion
config.repos = config.repos.map(repo => {
   (typeof repo == 'string') ? {name: repo} : repo;
 });
```